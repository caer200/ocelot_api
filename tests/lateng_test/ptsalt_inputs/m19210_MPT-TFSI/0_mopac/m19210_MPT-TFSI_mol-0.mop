CHARGES

 
H	4.611819 +1 4.208536 +1 4.763299 +1
C	5.332105 +1 4.260127 +1 5.380704 +1
C	5.375775 +1 5.314018 +1 6.311600 +1
C	6.313912 +1 3.306500 +1 5.353490 +1
C	6.430424 +1 5.296194 +1 7.261724 +1
N	4.429180 +1 6.329141 +1 6.275450 +1
H	6.272003 +1 2.616101 +1 4.702588 +1
C	7.372877 +1 3.326553 +1 6.257356 +1
C	7.420862 +1 4.304689 +1 7.211725 +1
S	6.621677 +1 6.465166 +1 8.501517 +1
C	4.321472 +1 7.340475 +1 7.224774 +1
C	3.531434 +1 6.381278 +1 5.111888 +1
H	8.054929 +1 2.666245 +1 6.211676 +1
H	8.129566 +1 4.313256 +1 7.844404 +1
C	5.252704 +1 7.483074 +1 8.274943 +1
C	3.263716 +1 8.261795 +1 7.168862 +1
H	3.307265 +1 7.314084 +1 4.913431 +1
H	2.710897 +1 5.882897 +1 5.309827 +1
H	3.977718 +1 5.980287 +1 4.336776 +1
C	5.131187 +1 8.524711 +1 9.195667 +1
H	2.611024 +1 8.185494 +1 6.482930 +1
C	3.160322 +1 9.271124 +1 8.090858 +1
H	5.773539 +1 8.616564 +1 9.889562 +1
C	4.095352 +1 9.415951 +1 9.108849 +1
H	2.435652 +1 9.882660 +1 8.033077 +1
H	4.016987 +1 10.125322 +1 9.736318 +1
