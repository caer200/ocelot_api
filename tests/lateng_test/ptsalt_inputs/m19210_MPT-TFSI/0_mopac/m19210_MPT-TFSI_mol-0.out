 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:10:00 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          4.61181900  *   4.20853600  *   4.76329900  *
     2       C          5.33210500  *   4.26012700  *   5.38070400  *
     3       C          5.37577500  *   5.31401800  *   6.31160000  *
     4       C          6.31391200  *   3.30650000  *   5.35349000  *
     5       C          6.43042400  *   5.29619400  *   7.26172400  *
     6       N          4.42918000  *   6.32914100  *   6.27545000  *
     7       H          6.27200300  *   2.61610100  *   4.70258800  *
     8       C          7.37287700  *   3.32655300  *   6.25735600  *
     9       C          7.42086200  *   4.30468900  *   7.21172500  *
    10       S          6.62167700  *   6.46516600  *   8.50151700  *
    11       C          4.32147200  *   7.34047500  *   7.22477400  *
    12       C          3.53143400  *   6.38127800  *   5.11188800  *
    13       H          8.05492900  *   2.66624500  *   6.21167600  *
    14       H          8.12956600  *   4.31325600  *   7.84440400  *
    15       C          5.25270400  *   7.48307400  *   8.27494300  *
    16       C          3.26371600  *   8.26179500  *   7.16886200  *
    17       H          3.30726500  *   7.31408400  *   4.91343100  *
    18       H          2.71089700  *   5.88289700  *   5.30982700  *
    19       H          3.97771800  *   5.98028700  *   4.33677600  *
    20       C          5.13118700  *   8.52471100  *   9.19566700  *
    21       H          2.61102400  *   8.18549400  *   6.48293000  *
    22       C          3.16032200  *   9.27112400  *   8.09085800  *
    23       H          5.77353900  *   8.61656400  *   9.88956200  *
    24       C          4.09535200  *   9.41595100  *   9.10884900  *
    25       H          2.43565200  *   9.88266000  *   8.03307700  *
    26       H          4.01698700  *  10.12532200  *   9.73631800  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          4.6118      4.2085      4.7633
     2         C          5.3321      4.2601      5.3807
     3         C          5.3758      5.3140      6.3116
     4         C          6.3139      3.3065      5.3535
     5         C          6.4304      5.2962      7.2617
     6         N          4.4292      6.3291      6.2755
     7         H          6.2720      2.6161      4.7026
     8         C          7.3729      3.3266      6.2574
     9         C          7.4209      4.3047      7.2117
    10         S          6.6217      6.4652      8.5015
    11         C          4.3215      7.3405      7.2248
    12         C          3.5314      6.3813      5.1119
    13         H          8.0549      2.6662      6.2117
    14         H          8.1296      4.3133      7.8444
    15         C          5.2527      7.4831      8.2749
    16         C          3.2637      8.2618      7.1689
    17         H          3.3073      7.3141      4.9134
    18         H          2.7109      5.8829      5.3098
    19         H          3.9777      5.9803      4.3368
    20         C          5.1312      8.5247      9.1957
    21         H          2.6110      8.1855      6.4829
    22         C          3.1603      9.2711      8.0909
    23         H          5.7735      8.6166      9.8896
    24         C          4.0954      9.4160      9.1088
    25         H          2.4357      9.8827      8.0331
    26         H          4.0170     10.1253      9.7363

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C13 H11 N S  =    26 atoms



      MOLECULAR POINT GROUP   :   Cs  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
